### PR TITLE
README update

### PR DESCRIPTION
**Pull request by akihikokuroda**
I added some information about the development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,39 @@
...
+   [fork of this repo](https://help.github.com/articles/fork-a-repo/)
+1. Clone it to your machine:
+
+```shell
+mkdir -p ${GOPATH}/src/github.com/openshift
+cd ${GOPATH}/src/github.com/openshift
+git clone git@github.com:${YOUR_GITHUB_USERNAME}/tektoncd-pipeline-operator.git
+cd tektoncd-pipeline-operator
+git remote add upstream git@github.com:tektoncd/tektoncd-pipeline-operator.git
+git remote set-url --push upstream no_push
```
Comment (sthaha) [inline README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -25,6 +53,27 @@ minikube start -p mk-tekton \
...
 
+### Development build
+
+1. change directory to '${GOPATH}/src/github.com/openshift/tektoncd-pipeline-operator'
+```
+cd ${GOPATH}/src/github.com/openshift/tektoncd-pipeline-operator
+```
+2. go build
+``` 
+CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o build/_output/bin/tekton-pipeline-operator ./cmd/manager/
```
Comment (sthaha) [inline README.md]: it is better to use the operator-sdk itself to do the build e.g. `operator-sdk build <image-name>`

Comment (akihikokuroda) [reply README.md]: Thanks for review.  I updated to use the operator-sdk

**Pull request merged**